app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    sepp_command = command.split()
+
+    source = sepp_command[1]
+    destination = sepp_command[2]
+
+    path_to_move = destination.split("/")
```

Comment:
Consider using `os.path.join` instead of manually concatenating paths with '/'. This will make your code cross-platform compatible.